tests/trinity/integration/test_lightchain_integration.py

```diff
@@ -33,31 +42,121 @@
 )
 
 
+@pytest.fixture(scope='module')
+def geth_port():
+    sock = socket.socket()
+    sock.bind(('127.0.0.1', 0))
+    port = sock.getsockname()[1]
+    sock.close()
+    return port
+
+
+@pytest.fixture(scope='module')
+def geth_datadir():
+    datadir = Path(__file__).parent / 'fixtures' / 'geth_lightchain_datadir'
+    return datadir.absolute()
+
+
+@pytest.fixture
+def enode(geth_port):
+    return (
+        "enode://"
+        "96b3d566ca9b9db43e4e5efa2e17fb95f7ddfe72981aadadda2080b4cab23c4d863acf31d715422cd87bec4faf3c8ad2e74da0065332d870b6bd07d0433bec71"  # noqa: E501
+        "@127.0.0.1:%d" % geth_port
+    )
+
+
+@pytest.fixture(scope='module')
+def geth_ipc_path(geth_datadir):
+    return geth_datadir / 'geth.ipc'
+
+
+@pytest.fixture(scope='module')
+def geth_binary():
+    path = os.environ.get('GETH_BINARY')
+    if not path:
+        raise EnvironmentError("GETH_BINARY must be set, with a path to the geth binary")
+    else:
+        return path
+
+
+@pytest.fixture(scope='module')
+def geth_command_arguments(geth_binary, geth_ipc_path, geth_datadir, geth_port):
+    return (
+        geth_binary,
+        '--testnet',
+        '--datadir', str(geth_datadir),
+        '--ipcpath', geth_ipc_path,
+        '--lightserv', '90',
+        '--nodiscover',
+        '--fakepow',
+        '--port', str(geth_port),
```

Comment:
Maybe we should bake these options into a config file so that we can also include our trinity node's enode in the `StaticNodes` or `TrustedNodes` configuration so that geth maintains a connection slot for our trinity node (which I don't think can be done via CLI arguments".

Reply:
I think we can skip that work for this test. This enode has never been connected to the network and has `--nodiscover` on. So AIUI, the node should never connect to any peers but ours.